luigi/worker.py

```diff
@@ -713,11 +713,25 @@ def _get_work_task_id(self, get_work_response):
         if get_work_response['task_id'] is not None:
             return get_work_response['task_id']
         elif 'batch_id' in get_work_response:
-            task = load_task(
-                module=get_work_response.get('task_module'),
-                task_name=get_work_response['task_family'],
-                params_str=get_work_response['task_params'],
-            )
+            try:
+                task = load_task(
+                    module=get_work_response.get('task_module'),
+                    task_name=get_work_response['task_family'],
+                    params_str=get_work_response['task_params'],
+                )
+            except Exception as ex:
```

Comment:
I used such a broad catch because there's no limit to what could go wrong here. batch_method can throw arbitrary errors because it can be an arbitrary function supplied by the user.